content/en/serverless/step_functions/merge-step-functions-lambda.md

```diff
@@ -178,9 +178,45 @@ To link your Step Function traces to nested Step Function traces, configure your
 }
 {{< /highlight >}}
 
-## Merge Lambda -> Step Functions -> Lambda
+## Merge upstream traces with Step Functions and downstream Lambda traces
```

Comment:
I think this title works!

Reply:
@avedmala (sorry for the late night comments).

Since this is the ideal configuration and it uses JSONata, can we make it the top section so essentially any customer who wants to do this can just use this option?

Reply:
@sumedham no worries! just moved it up and made some minor fixes